ui/scripts/network.js

```diff
@@ -6517,7 +6445,192 @@
                         }
                     }
                 }
-            }
+            },
+            vpnuser: {
+                        type: 'select',
+                        title: 'VPN Users',
+
+                        listView: {
+                        id: 'vpnUsers',
+                        fields: {
+                            username: { label: 'label.name' },
+                            domain: { label: 'label.domain' },
+                            state: { label: 'label.state',
+                                indicator: {
+                                    'Active': 'on',
+                                }
+                            }
+                        },
+                        dataProvider: function(args) {
+                            var data = {};
+                            listViewDataProvider(args, data);
+                            $.ajax({
+                                url: createURL('listVpnUsers'),
+                                data: data,
+                                dataType: 'json',
+                                async: true,
+                                success: function(data) {
+                                    args.response.success({
+                                        data: data.listvpnusersresponse.vpnuser
+                                    });
+                                }
+                            });
+                        },
+                        actions:{
+                            destroy: {
+                                label: 'label.action.delete.user',
+                                messages: {
+                                    confirm: function(args) { return 'Are you sure you want to delete the VPN user'},
```

Comment:
Here also you can replace this line ('Are you sure you want to delete the VPN user') with a localization key.